File: proyectomds2/src/main/java/vistas/VistaArtistaCancionMasReproducida.java
package vistas;

import com.vaadin.flow.templatemodel.TemplateModel;
import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.template.Id;
import com.vaadin.flow.component.html.Label;

/**
 * A Designer generated component for the vista-artista-cancion-mas-reproducida template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("vista-artista-cancion-mas-reproducida")
@JsModule("./src/vista-artista-cancion-mas-reproducida.js")
public class VistaArtistaCancionMasReproducida extends PolymerTemplate<VistaArtistaCancionMasReproducida.VistaArtistaCancionMasReproducidaModel> {

    @Id("label_top_canciones")
	private Label label_top_canciones;
	@Id("top_canciones1")
	private Element top_canciones1;
	@Id("lista_top_canciones")
	private Element lista_top_canciones;

	/**
     * Creates a new VistaArtistaCancionMasReproducida.
     */
    public VistaArtistaCancionMasReproducida() {
        // You can initialise any data required for the connected UI components here.
    }

    /**
     * This model binds properties between VistaArtistaCancionMasReproducida and vista-artista-cancion-mas-reproducida
     */
    public interface VistaArtistaCancionMasReproducidaModel extends TemplateModel {
        // Add setters and getters for template properties here.
    }
}

File: proyectomds2/frontend/src/vista-artista-cancion-mas-reproducida.js
import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';

class VistaArtistaCancionMasReproducida extends PolymerElement {

    static get template() {
        return html`
<style include="shared-styles">
                :host {
                    display: block;
                    height: 100%;
                }
            </style>
<vaadin-vertical-layout style="width: 100%; height: 100%;" id="top_canciones1">
 <label id="label_top_canciones">Top Canciones reproducidas </label>
 <vaadin-vertical-layout theme="spacing" style="width: 100%; height: 100%;" id="lista_top_canciones"></vaadin-vertical-layout>
</vaadin-vertical-layout>
`;
    }

    static get is() {
        return 'vista-artista-cancion-mas-reproducida';
    }

    static get properties() {
        return {
            // Declare your properties here.
        };
    }
}

customElements.define(VistaArtistaCancionMasReproducida.is, VistaArtistaCancionMasReproducida);
